src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.macios.cs

```diff
@@ -59,6 +59,22 @@ public override void ViewWillDisappear(bool animated)
 		base.ViewWillDisappear(animated);
 	}
 
+	/// <inheritdoc/>
+	public override void ViewWillTransitionToSize(CGSize toSize, IUIViewControllerTransitionCoordinator coordinator)
+	{
+		coordinator.AnimateAlongsideTransition((IUIViewControllerTransitionCoordinatorContext obj) => {
+			// Before screen rotate
+		}, (IUIViewControllerTransitionCoordinatorContext obj) => {
+			// After screen rotate
+			if (VirtualView is not null)
+			{
+				PopupExtensions.SetSize(this, VirtualView);
```

Comment:
Can we use it like extension method?

Reply:
@VladislavAntonyuk, Thank you for your review. 
Since it cannot be called directly within the target class where the extension method 
is defined, it was implemented to be called statically.